README.md

```diff
@@ -80,4 +80,50 @@ For openning pull requests you have to send PR's to [`develop`](https://github.c
         Accept: application/json; version=<your reuired version>
     
         example:
-            Accept: application/json; version=1.0.1
\ No newline at end of file
+            Accept: application/json; version=1.0.1
+
+
+
+## API Documentaion 
+Jobguy uses swagger to generate api docs
+
+you can see docs [here](https://api.jobguy.work/swagger/)
+
+
+
+if you head to api link, you'll see this page.
+
+this is the public API 
+![public api screen](./screenshots/public_api_screen.png)
+
+you can expand this parts and easily try them.
+
+jobguy has more than these public endpoints.
+
+if you want to access those endpoints, you need to authenticate.
+to do so, expand the **authnz** part,and you'll see something like this:
+![public api screen](./screenshots/auth_view_screen.png)
+
+### follow these steps for authentication 
+1. click on `/authnz/login_email/` endpoint
+2. click on Example value box on right side, so the required body parameters will appear in data input, and fill it with your own data.
+3. after you click on **Try it out!**, you'll get a response (like following image)
+
+![public api screen](./screenshots/token_included_response_screen.png)
+
+4. copy the token value, then click on `Authorize` button on top right of the screen, you'll see something like this:
+
+![public api screen](./screenshots/auth_pop_up_screen.png)
```

Comment:
I think you did it wrong, and it's not your fault you just don't know the rule. Because of that we want to add details of swagger to docs. :)
You should set value of authorization as `JWT your.jwt.token` test it yourself and you should see more access you have than a public user, then add it docs.

Reply:
oh!! i see my mistake